modules/objc/examples/code_snippets/GettingStarted.m

```diff
@@ -75,8 +89,9 @@ - (void)viewDidLoad {
         // Start replication
         [_replicator start];
 
+    } else {
+        NSLog(@"Not running replication");
```

Comment:
I don't think this is important to be in the example?

Reply:
This is to make the user look at the replication code and see that is optional, which will also point out the URL